pkg/ctl/create/cluster.go

```diff
@@ -319,8 +319,8 @@ func doCreateCluster(cmd *cmdutils.Cmd, ng *api.NodeGroup, params *cmdutils.Crea
 		if err != nil {
 			return err
 		}
-		tasks := stackManager.NewTasksToCreateClusterWithNodeGroups(cfg.NodeGroups, cfg.ManagedNodeGroups, supportsManagedNodes)
-		ctl.AppendExtraClusterConfigTasks(cfg, params.InstallWindowsVPCController, tasks)
```

Comment:
Was it intentional to lose this line?

Reply:
yeah because `AppendExtraCluster...` became `CreateExtraCluster...` and I call that before `NewTasksToCreate...`